lib/foreman_remote_execution/version.rb

```diff
@@ -1,3 +1,3 @@
 module ForemanRemoteExecution
-  VERSION = '1.4.5'.freeze
+  VERSION = '1.5.0'.freeze
```

Comment:
We do version bumps in separate commits, so you don't have to worry about it here.

Reply:
I was asked in https://github.com/theforeman/foreman_remote_execution/pull/130#issuecomment-362210308 😕

Reply:
@adamruzicka I know this is how it was done in past but I also remember people forgot to bump it later, so I'd prefer if we start bumping the version right away, unless there's some concern

Reply:
I don't feel all that strongly about this, as long as we don't double bump the version